catalog/dags/data_refresh/create_index.py

```diff
@@ -30,17 +31,14 @@ def create_index(
     # Generate a UUID suffix that will be used by the newly created index.
     temp_index_name = generate_index_name(media_type=data_refresh_config.media_type)
 
-    # Get the configuration for the new Elasticsearch index, based off the existing index.
-    index_config = es.get_index_configuration_copy.override(
-        task_id="get_index_configuration"
-    )(
-        source_index=data_refresh_config.media_type,
-        target_index_name=temp_index_name,
+    # Create a new index
+    es.create_index(
+        index_config={
+            "index": temp_index_name,
+            "body": index_settings(data_refresh_config.media_type),
```

Comment:
This change ensures that the data refresh always creates a new index based off the hard-coded index configuration in `es_mapping.py`, rather than by copying the configuration of the existing index.

This is how the ingestion server does it currently, and was just a miss in the previous PR. It's necessary to ensure that temporary or experimental changes to the indices (for example, created using our `create_new_es_index` DAGs or similar) are not carried forward in the data refresh without making the changes permanent via a code change/PR.